scripts/init_letsencrypt.sh

```diff
@@ -7,12 +7,12 @@ set -o allexport
 source .env
 set +o allexport
 
-CONFIG_PATH="${CONFIG_PATH:-'./conf'}"
+CONFIG_PATH="${CONFIG_PATH:-'./'}"
 
-if [ ! -e "docker-nginx/options-ssl-nginx.conf" ] || [ ! -e "docker-nginx/ssl-dhparams.pem" ]; then
+if [ ! -e "${CONFIG_PATH}/docker-nginx/options-ssl-nginx.conf" ] || [ ! -e "${CONFIG_PATH}/docker-nginx/ssl-dhparams.pem" ]; then
```

Comment:
in case of having .env in a parent repo it would either not find `.env` or the `./docker-nginx/...` depending on what location you run the script from...

Reply:
If I am not mistaken, the script is meant to be run from where you have your `.env` file.
So in the case of a parent repo with its `.env` file, you run it from there and the `CONFIG_PATH` variable allows to point to the correct path to find the `conf/` and `docker-nginx/` folders.